### PR TITLE
Metadata

### PR DESCRIPTION
**Pull request by DougManuel**
Expanded metadata with breaking change: `locationID` to `siteID`. 

We would need to update the change log, version and data if this change is accepted.

### CODE REVIEW

File: metadata.md [hunk 1]
```diff
@@ -1,35 +1,99 @@
...
-- **sampleType**: Type of wastewater sample: 
-  - sludge - 
-- **catchmentPop**:	Number of people represented in the wastewater sample.
-- **assayMethod**: Method used to assay viral signals.
-- **creator**:	Person 
-- **creatorEmail**:	
-- **maintainer**:	
-- **description**:
-- **notes**:
+- **siteID**: Identifier for location where wastewater sample was taken. Additional location information in `wastewater_site.csv`.
```
Comment (DougManuel) [inline metadata.md:23]: Note change in metadata from `locationID` to `siteID`.

File: metadata.md [hunk 2]
```diff
@@ -1,35 +1,99 @@
...
+- **sampleVariance**: Sample variance Ct/Cq.
+- **sampleMeanNormal**: Sample mean normalizing Ct/Cq.
+- **sampleVarianceNormal**: Sample variance normalizing Ct/Cq.
+
+## Site (wastewater_site.csv) <span id="site"><span>
+
+Site of SARS-CoV-2 wastewater sampling, including how the sample was collected and stored. Each SARS-CoV-2 test result corresponds to one site.
+
+- **siteID**:	Unique identifier for the location where wastewater sample was taken. `locationID` is the same in `wastewater_covid-19.csv`. (Primary Key)
+- **siteName**:	Name corresponding to `siteID`. Location name could be a treatment plant, campus, institution or sewer location, etc.
```
Comment (DougManuel) [inline metadata.md:48]: Note change in name from `locationID` to `siteID`.

File: metadata.md (hunk 1 above)
Comment (DougManuel) [reply metadata.md:23]: Name change to use more commonly used term by Ottawa WW program. There is a wide variation in the term for 'site' or 'location' in WBE worldwide. Probably just need to make and decision and go with it.

Comment (wyusuf068) [reply metadata.md:23]: `siteID` works for me.

File: metadata.md [hunk 3]
```diff
@@ -1,35 +1,99 @@
...
+- **date**: Date information was provided or updated.
+
+
+## Entity Relationship Diagram <span id="erd"><span>
+
+![](img/ERD.svg)
+
+Comment on the ERD in [Lucidcharts](https://lucid.app/invitations/accept/781822fc-6ac5-4aa7-9023-323fd4b6b04f)
+
+## Changelog
```
Comment (wyusuf068) [inline metadata.md:94]: Shouldn't this be in the [README](https://github.com/Big-Life-Lab/covid-19-wastewater/blob/metadata/README.md#changelog)?

**Review by wyusuf068 — APPROVED** [review]
Looks good to me. Not sure if we should have a changelog in `metadata.md` when one exists in the README.

**Pull request merged**